src/less/InfoBarWidget.less

```diff
@@ -0,0 +1,149 @@
+@import 'mixins';
+
+@spinner-size: 12px;
+
+.ext-wwt-infoBarWidget {
+	position: relative;
+	display: flex;
+	z-index: 1;
+	width: 100%;
+	box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.25);
+	padding: 16px;
+	color: #ffffff;
+	.skin-vector & {
+		// This duplicates Vector, making sure that the
+		// bar stops at the edge of the content even though
+		// it is outside the content box itself.
+		.ltr& {
+			margin-left: 11em;
+		}
+		.rtl& {
+			margin-right: 11em;
+		}
+		width: calc( 100% - 11em - 2em );
+	}
+
+	&.ext-wwt-infoBarWidget-sticky {
+		position: fixed;
+		top: 0;
+	}
+
+	.oo-ui-iconElement-icon {
+		width: 1em;
+	}
+
+	.oo-ui-labelElement-label {
+		flex-grow: 2;
+		margin-left: 2em;
+		.rtl & {
+			margin-left: 0;
+			margin-right: 2em;
+		}
+	}
+
+	.ext-wwt-infoBarWidget-close.oo-ui-iconElement {
+		margin: 0 1em;
+		cursor: pointer;
+
+		&.oo-ui-flaggedElement-pending {
+			display: none;
+		}
+	}
+
+	&.oo-ui-flaggedElement-pending {
+		background-color: #2a4b8d;
+
+		.ext-wwt-infoBarWidget-spinner {
+			display: inline-block;
+			width: 50px;
+			white-space: nowrap;
+
+			.ext-wwt-infoBarWidget-spinner-bounce,
+			&:before,
+			&:after {
+				content: '';
+				background-color: #fff;
+				display: block;
+				float: left;
+				margin-top: @spinner-size / 2;
+				width: @spinner-size;
+				height: @spinner-size;
+				border-radius: 100%;
+				.animation( wwtBouncedelay 1600ms ease-in-out -160ms infinite both );
+			}
+
+			&:before {
+				margin-right: 4px;
+				.animation-delay( -330ms );
+			}
+
+			&:after {
```

Comment:
Pseudo-elements should have two colons, e.g. `&::after` https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements#Syntax (same with lines 62-63, 75)

Reply:
Ooh, interesting

```
Note: As a rule, double colons (::) should be used instead of a single colon (:). This distinguishes pseudo-classes from pseudo-elements. However, since this distinction was not present in older versions of the W3C spec, most browsers support both syntaxes for the original pseudo-elements.
```
That explains why I saw it either way, and both ways worked. I'll change it.